README.md

```diff
@@ -108,4 +108,5 @@ Tags are released to:
 
 ## Contributing
 
-Contributions are always welcome! Please refer to our [contributing guidelines](CONTRIBUTING.md).
\ No newline at end of file
+Contributions are always welcome! Please refer to our [contributing guidelines](CONTRIBUTING.md).
+
```

Comment:
Now there is an extra newline :sweat_smile: 

I'll leave it to Armand as he noticed it but should still be fine to merge, the typo fix is more important here.

Reply:
I've added the additional setting to VsCode to fix the newline thing, Thanks @bamarni for the review.